includes/widgets/common.php

```diff
@@ -981,6 +981,498 @@ protected function register_controls() {
 			]
 		);
 
+		$this->start_controls_tabs( '_tabs_positioning' );
+
+		foreach ( [ 'normal', 'hover' ] as $tab ) {
+			$hover_state = 'hover' === $tab ? ':hover' : '';
```

Comment:
Instead of $hover_state maybe just $state? (feels likes a bit weird to call it hover_state when the value is empty and then it's not related to the hover..).

Reply:
done